### PR TITLE
Create code base assets and style

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
1. Add font and image to assets
2. Fix styles

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+VITE_API_BASE_URL = https://6684bd0356e7503d1ae0ff6f.mockapi.io
```
Comment (thanhnguyen-agilityio) [inline .env.example]: Do not put sensitive data to source.

File: src/assets/images/banner.svg
Comment (thanhnguyen-agilityio) [inline src/assets/images/banner.svg:1]: Why this image is `svg`?

File: .env.example (hunk 1 above)
Comment (thanhtrantruong47) [reply .env.example]: fixed

File: src/assets/images/banner.svg
Comment (thanhtrantruong47) [reply src/assets/images/banner.svg:1]: fixed

**Review by hungdo-agilityio — APPROVED** [review]

File: src/styles/variable.css [hunk 2]
```diff
@@ -28,21 +28,21 @@
...
-  --border-radius: 10px;
+  --border-radius: 0.625rem; /* 10px */
 
-  /*Light height */
-  --line-height-sm: 19.2px;
-  --line-height-md: 24px;
-  --line-height-lg: 30px;
-  --line-height-xl: 75px;
+  /* Line height */
+  --line-height-sm: 1.2rem; /* 19.2px */
```
Comment (thanhnguyen-agilityio) [inline src/styles/variable.css:39]: I don't think these line-height correct, let show me the image where you will use this values.

File: src/styles/variable.css [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
   /* Font Sizes */
-  --font-size-sm: 14px;
-  --font-size-md: 18px;
-  --font-size-lg: 36px;
-  --size-xl: 40px;
-  --font-size-2xl: 48px;
+  --font-size-sm: 0.875rem; /* 14px */
+  --font-size-md: 1.125rem; /* 18px */
+  --font-size-lg: 2.25rem; /* 36px */
+  --size-xl: 2.5rem; /* 40px */
```
Comment (thanhnguyen-agilityio) [inline src/styles/variable.css]: Is the `--size-xl` not have correct naming?

File: src/styles/variable.css [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+  --font-size-2xl: 3rem; /* 48px */
 
   /* Font Weights */
   --font-weight-regular: 400;
```
Comment (thanhnguyen-agilityio) [inline src/styles/variable.css:10]: `--font-weight-regular` -> `--fw-regular`

File: src/styles/variable.css (hunk 2 above)
Comment (thanhtrantruong47) [reply src/styles/variable.css:39]: image 
image
image
image
image
image

File: src/styles/variable.css (hunk 3 above)
Comment (thanhtrantruong47) [reply src/styles/variable.css]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**